client/material/button.tsx

```diff
@@ -285,12 +290,12 @@ export interface HotkeyedTextProps {
  * given hotkey. Technically, this component is not really button-specific, but that's mostly where
  * it will be used so we leave it here.
  */
-export const HotkeyedText = React.memo(({ hotkey, text, disabled }: HotkeyedTextProps) => {
+export const HotkeyedText = React.memo(({ hotkeys, text, disabled }: HotkeyedTextProps) => {
   if (disabled) {
     return <>{text}</>
   }
 
-  const hotkeyString = String.fromCharCode(hotkey.keyCode).toLowerCase()
+  const hotkeyString = String.fromCharCode(hotkeys[0].keyCode).toLowerCase()
```

Comment:
It doesn't really make sense to me to do this here, just accept a single hotkey and let the component using it decide which hotkey that is